docs/content/modeling/public-access.mdx

```diff
@@ -115,18 +115,31 @@ Let us create a relationship tuple that states: **any user can view document:com
   ]}
 />
 
+:::caution Wildcard syntax usage
+Please note that `*` is a special <ProductName format={ProductNameFormat.ShortForm}/> syntax meaning **everyone** when used as a _user_ within a _relationship tuple_. It is not a wildcard or regex expression.
+
+**You cannot use the `*` syntax in the tuple's object field.**
+
+The following syntax is invalid:
+<WriteRequestViewer
+  relationshipTuples={[
+    {
+      _description: 'Not allow to specify * in the object field',
+      user: 'user:*',
+      relation: 'view',
+      object: 'document:*',
+    },
+  ]}
+/>
+
```

Comment:
**You cannot use the this syntax as part of a userset in the tuple's user field.**

The following syntax is invalid:
```jsx
<WriteRequestViewer
  relationshipTuples={[
    {
      _description: 'It is invalid to use this syntax as part of a userset. The below relationship tuple is invalid and does not mean that members of any org can view the Getting Started document.',
      user: 'org:*#member',
      relation: 'can_view',
      object: 'document:getting-started',
    },
  ]}
/>
```

Reply:
Added